learing_journal/templates/edit-entry.jinja2

```diff
@@ -1,15 +1,24 @@
-{% extends "layout_new_entry.jinja2" %}
+{% extends "layout.jinja2" %}
 {% block body %}
 
 <div class="jumbotron">
-  <!-- Entry ID: {{ single_entry.id }} -->
-  <h3>edit-entry.jinja2</h3>
-  <h2>Edit Journal Entry {{ single_entry.id }}</h2>
+  <h1>edit-entry.jinja2</h1>
+  <!-- TODO: Wrap the back to home link in a button -->
+  <!-- <h2><a href='/'>Back to Home</a></h2> -->
```

Comment:
Alright well don't forget to do this. Or even easier: put a button in your layout as a part of your site's menu that way you don't have to do it on every page.